multimedia/scrcpy/Portfile

```diff
@@ -4,8 +4,9 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           meson 1.0
 PortGroup           compiler_blacklist_versions 1.0
+PortGroup           legacysupport 1.1
 
-github.setup        Genymobile scrcpy 2.1 v
+github.setup        Genymobile scrcpy 2.1.1 v
```

Comment:
I know this isn't related to your change... but, please add `github.tarball_from ??` and specify either `releases` (preferred if upstream provides these) or set it to `archive`. In both cases, this will likely require you to update the `checksums` as well.

Reply:
Addressed in https://github.com/macports/macports-ports/pull/20727/commits/fccab39fc1ba10b17275c8208f29723316493073.